marklogic-data-hub-central/src/main/resources/explore-data/ml-search-lib/search.sjs

```diff
@@ -140,6 +140,18 @@ class Search {
     return recordCount;
   }
 
+  getMetrics(metricTypes) {
```

Comment:
It looks like you are returning the same data every time. For What's New, we want to allow the user to select different time ranges and see the values change:
<img width="446" alt="Screen Shot 2022-04-22 at 10 51 36 AM" src="https://user-images.githubusercontent.com/477757/164768589-e434f400-f3e0-41a3-ad11-f30659ad2c95.png">
I think the best way to mock this is for you to return random numbers for all the values. To return numbers from 1000 to 100,000 in JavaScript:
```
Math.floor(Math.random() * (100000 - 1000 + 1)) + 1000;
```
Can we do that for all the values?